style-transfer/styleTransfer.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+function sizeOfShape(shape) {
```

Comment:
Please help consolidate this function with other examples. Probably, we can have a common utils.js.